stripe/src/main/java/com/stripe/android/StripePaymentController.kt

```diff
@@ -1148,6 +1203,23 @@ internal class StripePaymentController internal constructor(
                 )
             )
         }
+
+        /**
+         * Calculate a linear backoff delay before retrying the next completion request.
```

Comment:
probably better to use [exponential backoff](https://en.wikipedia.org/wiki/Exponential_backoff)? i.e 
* delay 2^1-1 = 1 seconds before the first retry
* delay 2^2-1 = 3 seconds before the 2nd retry
* delay 2^3-1 = 7 seconds before the 3nd retry